### ISSUE TITLE
Add new TEPs to teps-service.js

### ISSUE THREAD

**Issue by ireneisdoomed** [opentargets/platform#1384]
We have annotated 9 new TEPs, which need to be added to [tips-service.js ](https://github.com/opentargets/webapp/blob/master/app/src/services/teps-service.js). We are in the process of automatising this annotation, could you @LucaFumis or @andrewhercules add them in any of these ways?:

1. either you can copy paste the generated JSON with all the current TEPs right from [this file](https://github.com/opentargets/platform/files/5954024/tep_annotation-2021-02-08.json.gz);

2. or add the new ones as we have been doing before:
```
"ENSG00000124067": {
        "id": "ENSG00000124067",
        "symbol": "SLC12A4",
        "link": "https://www.thesgc.org/tep/slc12a4slc12a6",
        "disease": "Sickle cell disease (SCD), Neurological",
        "uniprot_id": "Q9UP95"
    },
    "ENSG00000082397": {
        "id": "ENSG00000082397",
        "symbol": "EPB41L3",
        "link": "https://www.thesgc.org/tep/epb41l3",
        "disease": "Neurodegeneration",
        "uniprot_id": "Q9Y2J2"
    },
    "ENSG00000168918": {
        "id": "ENSG00000168918",
        "symbol": "INPP5D",
        "link": "https://www.thesgc.org/tep/inpp5d",
        "disease": "Neurological Disorders",
        "uniprot_id": "Q92835"
    },
    "ENSG00000147065": {
        "id": "ENSG00000147065",
        "symbol": "MSN",
        "link": "https://www.thesgc.org/tep/moesin",
        "disease": "Alzheimer's disease",
        "uniprot_id": "P26038"
    },
    "ENSG00000041982": {
        "id": "ENSG00000041982",
        "symbol": "TNC",
        "link": "https://www.thesgc.org/tep/hfbg-c",
        "disease": "Inflammatory diseases",
        "uniprot_id": "P24821"
    },
    "ENSG00000164181": {
        "id": "ENSG00000164181",
        "symbol": "ELOVL7",
        "link": "https://www.thesgc.org/tep/elovl7",
        "disease": "Metabolic diseases",
        "uniprot_id": "A1L3X0"
    },
    "ENSG00000181192": {
        "id": "ENSG00000181192",
        "symbol": "DHTKD1",
        "link": "https://www.thesgc.org/tep/dhtkd1",
        "disease": "Metabolic diseases",
        "uniprot_id": "Q96HY7"
    },
    "ENSG00000152457": {
        "id": "ENSG00000152457",
        "symbol": "DCLRE1C",
        "link": "https://www.thesgc.org/tep/dclre1c",
        "disease": "Oncology",
        "uniprot_id": "Q96SD1"
    },
    "ENSG00000164458": {
        "id": "ENSG00000164458",
        "symbol": "TBXT",
        "link": "https://www.thesgc.org/tep/tbxt",
        "disease": "Cancer",
        "uniprot_id": "J3KP65"
    }
```

Please note that we now include 2 new fields in the annotation: `disease` and `uniprot_id`. Please let me know if this could cause any problems.

**Comment by andrewhercules** [comment]
Thank you @ireneisdoomed - will add to the teps-services.js file!

**Comment by andrewhercules** [comment]
Ticket closed as work completed and merged into 3.21.0 release
